integration_tests/integration/choose-practitioner.spec.ts

```diff
@@ -325,4 +325,80 @@ context('Choose Practitioner', () => {
       choosePractitionerPage.radio('N03F02::OM2').should('not.exist')
     })
   })
+
+  it('should not show team which does not exist in probation estate but is in user preferences', () => {
+    cy.task('stubUserPreferenceTeams', ['TM1', 'OLDTEAM1'])
+    cy.task('stubGetUnallocatedCasesByTeams', {
+      teamCodes: 'TM1,OLDTEAM1',
+      response: [
+        {
+          teamCode: 'TM1',
+          caseCount: 4,
+        },
+      ],
+    })
+    cy.task('stubWorkloadCases', {
+      teamCodes: 'TM1,OLDTEAM1',
+      response: [
+        {
+          teamCode: 'TM1',
+          totalCases: 3,
+          workload: 77,
+        },
+      ],
+    })
+    cy.task('stubGetTeamsByCodes', {
+      codes: 'TM1,OLDTEAM1',
+      response: [
+        {
+          code: 'TM1',
+          name: 'Team 1',
+        },
+      ],
+    })
+    cy.task('stubChoosePractitioners', {
+      teamCodes: ['TM1', 'OLDTEAM1'],
+      crn: 'J678910',
+      teams: {
+        TM1: [
+          {
+            code: 'OM1',
+            name: {
+              forename: 'Jane',
+              middleName: '',
+              surname: 'Doe',
+            },
+            email: 'j.doe@email.co.uk',
+            grade: 'PQiP',
+            workload: 19,
+            casesPastWeek: 2,
+            communityCases: 3,
+            custodyCases: 5,
+          },
+        ],
+        OLDTEAM1: [
```

Comment:
So there is still a practitioner associated with the old team?

Reply:
yeah there was, I made the request with Wisbech staff code and there were still practitioners in there